services/net/contentmigration/migrators/PaperMigrator.cs

```diff
@@ -115,6 +121,42 @@ public override SourceContent CreateSourceContent(LookupModel lookups, SourceMod
         return content;
     }
 
+    /// <summary>
+    /// fix the replace of paragraph markers with "|"
+    /// </summary>
+    /// <param name="text"></param>
+    /// <param name="summary"></param>
+    /// <returns>formatted body string</returns>
+    private string GetNewsItemBody(string text, string summary)
+    {
+        string body = string.Empty;
+
+        if (!string.IsNullOrEmpty(text))
+        {
+            string const marker = '|';
+            int index = text.IndexOf(marker);
```

Comment:
Don't change but this seems like something that would be nice to use configuration for.  If there are other special characters showing up, we can modify to handle with configuration instead of a code change.